config/webpack.config.prod.js

```diff
@@ -35,9 +36,7 @@ module.exports = {
     path: buildPath,
     filename: '[name].[chunkhash].js',
     chunkFilename: '[name].[chunkhash].chunk.js',
-    // TODO: this wouldn't work for e.g. GH Pages.
-    // Good news: we can infer it from package.json :-)
-    publicPath: '/'
+    publicPath: publicPath
```

Comment:
Yeah, agreed - good call. Putting together a regex that could do it, but do you think there is a more elegant way?